pkg/organization/router_test.go

```diff
@@ -0,0 +1,146 @@
+// Copyright 2020 The Moov Authors
+// Use of this source code is governed by an Apache License
+// license that can be found in the LICENSE file.
+
+package organization
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/antihax/optional"
+	"github.com/gorilla/mux"
+	"github.com/moov-io/base"
+	"github.com/moov-io/paygate/pkg/client"
+	"github.com/stretchr/testify/require"
+)
+
+var (
+	orgRepo = &MockRepository{
+		Config: &client.OrganizationConfiguration{
+			CompanyIdentification: base.ID(),
+		},
+	}
+)
+
+func TestGetOrganizationConfig(t *testing.T) {
+	req := httptest.NewRequest("GET", "/configuration/transfers", nil)
+	req.Header.Set("X-Organization", "moov")
+	w := httptest.NewRecorder()
+
+	router := mux.NewRouter()
+	NewRouter(orgRepo).RegisterRoutes(router)
+	router.ServeHTTP(w, req)
+	w.Flush()
+
+	require.Equal(t, http.StatusOK, w.Code)
+
+	var response client.OrganizationConfiguration
+	if err := json.NewDecoder(w.Body).Decode(&response); err != nil {
+		t.Fatal(err)
+	}
+	if response.CompanyIdentification == "" {
```

Comment:
Nit: You could swap statements like this for `require.NotEmpty()` or `assert.NotEmpty()`. (`require` will fail the test _now_, `assert` fails the test, but allows it to keep running.)

Reply:
good to know.